tests/e2e/util.go

```diff
@@ -834,9 +834,9 @@ func waitVCenterServiceToBeInState(serviceName string, host string, state string
 	return waitErr
 }
 
-//httpGet takes client and http Request as input and performs GET operation
+//httpDo takes client and http Request as input and performs the requested operation
 // and returns bodybytes
-func httpGet(client *http.Client, req *http.Request) []byte {
+func httpDo(client *http.Client, req *http.Request) []byte {
```

Comment:
PR 1309 and this PR will get merge conflits -
Since you modify the httpGet to Do  - the scaleTKGWorker & getGC functions will have to be merged.